### PR TITLE
3 VolumeProfile drawing tools and AnchoredVWAP

### PR DESCRIPTION
**Pull request by sbgtrading**
This pull request is only for VolumeProfileComposite, VolumeProfileManual and VolumeProfileRealtime, and it is also for AnchoredVWAP.

### CODE REVIEW

File: Tickblaze.Scripts/Drawings/AnchoredVWAP.cs [hunk 1]
```diff
@@ -1,132 +1,206 @@
-﻿namespace Tickblaze.Scripts.Drawings;
+﻿
+using Tickblaze.Scripts.Indicators;
 
-public abstract class AnchoredVWAP : Drawing
+namespace Tickblaze.Scripts.Drawings;
+
+public sealed class AnchoredVWAP : Drawing
```
[the remark this answers is not in the document]
Comment (sbgtrading) [reply Tickblaze.Scripts/Drawings/AnchoredVWAP.cs:3]: That was my attempt to exclude AnchoredVWAP from this branch.

[the remark this answers is not in the document]
Comment (sbgtrading) [reply Tickblaze.Scripts/Drawings/AnchoredVWAP.cs:3]: Before we change the file name and class, we would need to verify that with Jonathan and Sean.  Sean was upset with long names of several existing indicators (like MovingAverageConvergenceDivergence).  I believe Sean wants the name to be AnchoredVWAP.  I would then suggest we leave the file name and the class name as that, for consistency.

[the remark this answers is not in the document]
Comment (whis) [reply Tickblaze.Scripts/Drawings/AnchoredVWAP.cs:3]: File name of the source code is not equal to the display name displayed in the platform, that is defined by the `Name` property.

[the remark this answers is not in the document]
Comment (sbgtrading) [reply Tickblaze.Scripts/Drawings/AnchoredVWAP.cs]: The file names and class names have been changed as you requested

**Review by whis — APPROVED** [review]

**Pull request merged**